routers/web/repo/issue.go

```diff
@@ -133,7 +133,7 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 
 	var (
 		assigneeID        = ctx.FormInt64("assignee")
-		posterID          int64
+		posterID          = ctx.FormInt64("poster")
```

Comment:
This creates a conflict with the "created by me" view type, but there's already a conflict between assigneeID and the "assigned to me" view type, so I'm assuming that's ignorable for now. If it would make more sense to remove the "created by me" view type, though, I can make that change.